spec/oppiabotSpec.js

```diff
@@ -0,0 +1,173 @@
+          params: {
+            number: 5139,
+            owner: 'oppia',
+            repo: 'oppia',
+            body: 'Hi! @tester7777. Welcome to Oppia! ' +
+              'Please could you follow the instructions' +
+              '<a href=\"https://github.com/oppia/oppia/wiki/Contributing-code-to-Oppia#setting-things-up\">here</a>' +
+              "to get started ? You'll need to do this before" +
+              'we can accept your PR. Thanks!'
+          }
+        }),
+        getIssueLabels: jasmine.createSpy('getIssueLabels').and.returnValue({
+          data:
+            [
+              {
+                id: 248679580,
+                node_id: 'MDU6TGFiZWwyNDg2Nzk1ODA=',
+                url: 'https://api.github.com/repos/oppia/oppia/labels/PR:%20LGTM',
+                name: 'PR: LGTM',
+                color: '009800',
+                'default': false
+              },
+              {
+                id: 971968901,
+                node_id: 'MDU6TGFiZWw5NzE5Njg5MDE=',
+                url: "https://api.github.com/repos/oppia/oppia/labels/PR:%20don't%20merge%20-%20HAS%20MERGE%20CONFLICTS",
+                name: "PR: don't merge - HAS MERGE CONFLICTS",
+                color: 'd93f0b',
+                'default': false
+              },
+              {
+                id: 638838928,
+                node_id: 'MDU6TGFiZWw2Mzg4Mzg5Mjg=',
+                url: 'https://api.github.com/repos/oppia/oppia/labels/PR:%20for%20current%20release',
+                name: 'PR: for current release',
+                color: 'FF69B4',
+                'default': false
+              },
+              {
+                id: 638839900,
+                node_id: 'MDU6TGFiZWw2Mzg4Mzk5MDA=',
+                url: 'https://api.github.com/repos/oppia/oppia/labels/PR:%20released',
+                name: 'PR: released',
+                color: '00FF00',
+                'default': false
+              }
+            ]
+        }),
+        removeLabel: jasmine.createSpy('removeLabel')
+      }
+    };
+    robot.auth = () => Promise.resolve(github);
+    done();
+  });
+
+  describe('apiForSheets', () => {
+    beforeEach(function (done) {
+      spyOn(apiForSheetsModule, 'apiForSheets').and.callThrough();
+      spyOn(apiForSheetsModule, 'authorize').and.callThrough();
+      spyOn(apiForSheetsModule, 'checkClaSheet').and.callThrough();
+      robot.receive(pullRequestpayload);
+      done();
+    });
+
+    it('should be called for the given payload', () => {
+      expect(apiForSheetsModule.apiForSheets).toHaveBeenCalled();
+    });
+
+    it('should be called once for the given payload', () => {
+      expect(apiForSheetsModule.apiForSheets.calls.count()).toEqual(1);
+    });
+
+    it('should be called with three arguments for the given payload', () => {
+      expect(
+        apiForSheetsModule.apiForSheets.calls.argsFor(0).length).toEqual(3);
+    });
+
+    it('should be called with the' +
```

Comment:
try to fit as many words as you can in the first line and also you'll have to add an extra space at the end of the last word otherwise it'll appear as `thecorrect` instead of `the correct`

Reply:
Done, thanks!